File: Type-3/JoseVitorSAmorim_1.c
#include <stdio.h>

int main() {
    char nome[100];
    float nota;
    int presenca;
    
    printf("Nome: ");
    scanf(" %99[^\n]", nome);
    printf("Nota: ");
    scanf("%f", &nota);
    printf("Presenca(%%): ");
    scanf("%d", &presenca);
    if (presenca >= 50 && nota >= 80) {
        printf("Aluno %s aprovado\n", nome);
    } 
    else if (presenca >= 75 && nota >= 60) {
        printf("Aluno %s aprovado\n", nome);
    } 
    else {
        printf("Aluno %s reprovado\n", nome);
    }

    return 0;
}

File: Desen_Sis&Back-End/Type-3/JoseVitorSAmorim_1.java
import java.util.Scanner;
public class JoseVitorSAmorim_1 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Nome: ");
        String nome = scanner.nextLine();
        System.out.print("Nota: ");
        float nota = scanner.nextFloat();
        System.out.print("Presenca(%): ");
        int presenca = scanner.nextInt();
        if (presenca >= 50 && nota >= 80) {
            System.out.println("Aluno " + nome + " aprovado");
        } 
        else if (presenca >= 75 && nota >= 60) {
            System.out.println("Aluno " + nome + " aprovado");
        } 
        else {
            System.out.println("Aluno " + nome + " reprovado");
        }
    }
}
